src/components/SearchButton/SearchButton.module.css

```diff
@@ -0,0 +1,11 @@
+/*------------------------------------*\
+    # SEARCH BUTTON
+\*------------------------------------*/
+
+/**
+ * Button to trigger search in a Search Bar.
+ */
+.search-button {
+  /* Matches height with input field */
+  height: calc(var(--eds-size-5) + 2 * var(--eds-theme-form-border-width));
```

Comment:
Could we use `box-sizing: content-box` instead of `+ 2 * var(--eds-theme-form-border-width))`?

Reply:
Didn't work but `height: auto` did!